### ISSUE TITLE
GPS aquisition affecting landscape.

### ISSUE THREAD

**Issue by infinumimage** [Stellarium/stellarium#3514]
Just updated to 23.3 a couple of days ago and found an odd problem. 
I use a Globalsat BU-353N5 GPS receiver, it's never been a problem before however within about 10 seconds of starting the location aquisition my landscape disappeared off the screen and would keep reverting to the "zero horizon" whenever I tried to reapply my custom landscape. 
Tried it with a few of the other preinstalled ones and the exact same thing, wouldn't stay and kept disappearing and showing it as zero horizon landscape.

After reinstalling it twice I gave up the other night but looking at it again today it's only a problem while pulling down GPS data, it stops once I switch the GPS off and I can select a chosen landscape again. 
[log.txt](https://github.com/Stellarium/stellarium/files/13400550/log.txt)

Log file attached, pretty easy to recreate it so can possibly do a screen recording if needed.

MSI GF76 Katana
W11 64bit 23H2
Intel i7 12700H
RTX3050Ti  v546.01 Studio drivers
16GB RAM
Stellarium 23.3.0

Cheers

**Comment by gzotti** [comment]
Yes, it is assumed that when retrieving new coordinates, the landscape is invalid. When you click on the map, we load a zero horizon with color from the clicked point.

Did you try disabling "set landscapes automatically" (or similar)?

**Comment by infinumimage** [comment]
Hi Georg
I've set my custom landscape to load as default but it's the first time Ive seen it do this, it has never been an issue in previous versions.
Think I was on v23.1 prior to this one, the last time I used it i was away from home and this was the first time using 23.3 so I needed to put my home coordinates back in the the scope and NINA with the receiver.

I can live with it, not the kind of issue that will kill a nights imaging and I probably won't need to plug it in again for some time so should be a problem.

Regards

Andrew.
